### PR TITLE
Fix add_sequence

### LINKED ISSUE #5306: addSequence not in JVM
Easy-peasy to reproduce:

```
rg = hl.get_reference('GRCh38')
rg.add_sequence('gs://hail-common/references/Homo_sapiens_assembly38.fasta.gz',
                'gs://hail-common/references/Homo_sapiens_assembly38.fasta.fai')
```
throws:
```
Py4JError: is.hail.variant.ReferenceGenome.addSequence does not exist in the JVM
```

### PR DESCRIPTION
**Pull request by tpoterba**
fixes #5306

### CODE REVIEW

**Comment by tpoterba** [comment]
The bigger problem is that this functionality isn't tested. Do we have a tiny dummy reference we can make a fasta for?

**Comment by jigold** [comment]
I'll try and make a small fasta file

**Comment by jigold** [comment]
It's already there: src/test/resources/fake_reference.fasta

**Comment by jigold** [comment]
It looks like this test just needed to be modified:

```
    def test_reference_genome_sequence(self):
        gr3 = ReferenceGenome.read(resource("fake_ref_genome.json"))
        self.assertEqual(gr3.name, "my_reference_genome")
        self.assertFalse(gr3.has_sequence())

        gr4 = ReferenceGenome.from_fasta_file("test_rg", resource("fake_reference.fasta"),
                                              resource("fake_reference.fasta.fai"),
                                              mt_contigs=["b", "c"], x_contigs=["a"])
        self.assertTrue(gr4.has_sequence())
        self.assertTrue(gr4.x_contigs == ["a"])

        t = hl.import_table(resource("fake_reference.tsv"), impute=True)
        self.assertTrue(hl.eval(t.all(hl.get_sequence(t.contig, t.pos, reference_genome=gr4) == t.base)))

        l = hl.locus("a", 7, gr4)
        self.assertTrue(hl.eval(l.sequence_context(before=3, after=3) == "TTTCGAA"))
```

File: hail/python/hail/backend/backend.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def get_reference(self, name):
         return json.loads(Env.hail().variant.ReferenceGenome.getReference(name).toJSONString())
 
     def add_sequence(self, name, fasta_file, index_file):
-        Env.hail().variant.ReferenceGenome.addSequence(name, fasta_file, index_file)
+        scala_object(Env.hail().variant, 'ReferenceGenome').addSequence(name, fasta_file, index_file)
```
Comment (jigold) [inline hail/python/hail/backend/backend.py]: Why didn't this affect `remove_sequence` etc?

Comment (tpoterba) [reply hail/python/hail/backend/backend.py]: same issue!

Comment (jigold) [reply hail/python/hail/backend/backend.py]: What about `add_liftover` below? I'm confused why some of this seems to work but not others.

**Pull request merged**